yarn.lock

```diff
@@ -3817,6 +3817,35 @@ jsdom@^11.4.0:
     whatwg-url "^6.3.0"
     xml-name-validator "^2.0.1"
 
+jsdom@^11.5.1:
```

Comment:
should be deduped together with the above...

Reply:
good point! 🙂 fixed